### DISCUSSION TITLE
How does name in ins_seg_val.json of Instance Segmentation correspond to the image in val images folder?

### DISCUSSION THREAD

**Discussion by lihongyong** [bdd100k/bdd100k#258] [Q&A]
How does name in ins_seg_val.json of Instance Segmentation correspond to the image in val images folder?
for example，ins_seg_val.json has an image whose "name" is "9d824e04-f4274Fed.jpg", but I can't find this image in the Val Images Folder. How does it correspond?

**Comment by iremonur** [comment]
Hi @lihongyong, I have encountered the same issue, did you solve it?
